build-tests/api-documenter-test/etc/yaml/api-documenter-test/docclass1.yml

```diff
@@ -154,9 +155,57 @@ items:
       return:
         type:
           - 'api-documenter-test!SystemEvent:class'
+  - uid: 'api-documenter-test!DocClass1#seeExample:member(1)'
+    summary: Example of a `@see` tag.
+    seealsoContent: |-
+      -  [IDocInterface5](xref:api-documenter-test!IDocInterface5:interface) for the returned data structure
+         
+      -  your developer SDK...
+         
+         ...for
+         
+         ```
+         code
+         
+         ```
+         samples
+         
+      -  [RFC 1738](https://tools.ietf.org/html/rfc1738) for syntax
+    name: seeExample()
+    fullName: seeExample()
+    langs:
+      - typeScript
+    type: method
+    syntax:
+      content: 'seeExample(): void;'
+      return:
+        type:
+          - void
+        description: ''
   - uid: 'api-documenter-test!DocClass1.sumWithExample:member(1)'
     summary: Returns the sum of two numbers.
     remarks: This illustrates usage of the `@example` block tag.
+    examples:

```

Comment:
@rbuckton Does this DocFX field look correct to you? I don't have an easy way to test it.